src/modules/tests/tests.effects.ts

```diff
@@ -36,4 +40,60 @@ export class TestsEffects {
     )),
   );
 
+  @Effect()
+  startPracticeTestEffect$ = this.actions$.pipe(
+    ofType(testActions.START_PRACTICE_TEST),
+    switchMap(() => {
+      const slotData = {
+        ...practiceSlot,
+      };
+
+      return [
+        new PopulateApplicationReference(slotData.booking.application),
+        new PopulateCandidateDetails(slotData.booking.candidate),
+      ];
+    }),
+  );
+
 }
+
+export const application: Application = {
```

Comment:
Would these be better placed in a `/mocks` folder?

Reply:
Yeah, I totally couldn't decide the best place for them. We don't have a mocks folder yet do we?

Reply:
Ah, `/__mocks__` you mean?

Reply:
sorry yes that's it